challenges-webapp/src/main/webapp/vue-app/engagement-center/components/EngagementCenter.vue

```diff
@@ -44,6 +48,21 @@ export default {
   data: () => ({
     engagementCenterEnabled: eXo.env.portal.engagementCenterEnabled,
     tab: null,
+    isAdministrator: false,
   }),
+  created() {
+    this.isAdministratorCall();
```

Comment:
the method name that starts with `is` should be used for boolean only, you may use `init` method name by example

Reply:
done !